content/en/docs/ambient/install/multicluster/multi-primary_multi-network/index.md

```diff
@@ -0,0 +1,412 @@
+  namespace: istio-system
+  labels:
+    topology.istio.io/network: "network1"
+spec:
+  gatewayClassName: istio-east-west
+  listeners:
+  - name: mesh
+    port: 15008
+    protocol: HBONE
+    tls:
+      mode: Terminate # represents double-HBONE
+      options:
+        gateway.istio.io/tls-terminate-mode: ISTIO_MUTUAL 
+EOF
+{{< /text >}}
+
+{{< warning >}}
+TODO: Handle revisions
+{{< /warning >}}
+
+Apply the configuration to `cluster1`:
+
+{{< text bash >}}
+$ istioctl install --context="${CTX_CLUSTER1}" -f cluster1-ewgateway.yaml
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+Wait for the east-west gateway to be assigned an external IP address:
+
+{{< text bash >}}
+$ kubectl --context="${CTX_CLUSTER1}" get svc istio-eastwestgateway -n istio-system
+NAME                    TYPE           CLUSTER-IP    EXTERNAL-IP    PORT(S)   AGE
+istio-eastwestgateway   LoadBalancer   10.80.6.124   34.75.71.237   ...       51s
+{{< /text >}}
+
+## Expose services in `cluster1`
+
+TODO: change to explanation about service/namespace scope
+
+## Set the default network for `cluster2`
+
+If the istio-system namespace is already created, we need to set the cluster's network there:
+
+{{< text bash >}}
+$ kubectl --context="${CTX_CLUSTER2}" get namespace istio-system && \
+  kubectl --context="${CTX_CLUSTER2}" label namespace istio-system topology.istio.io/network=network2
+{{< /text >}}
+
+## Configure cluster2 as a primary
+
+Create the `istioctl` configuration for `cluster2`:
+
+{{< tabset category-name="multicluster-install-type-cluster-2" >}}
+
+{{< tab name="IstioOperator" category-value="iop" >}}
+
+Install Istio as primary in `cluster2` using istioctl and the `IstioOperator` API.
+
+{{< text bash >}}
+$ cat <<EOF > cluster2.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: mesh1
+      multiCluster:
+        clusterName: cluster2
+      network: network2
+    profile: ambient
+    pilot:
+      env:
+        AMBIENT_ENABLE_MULTI_NETWORK: "true"
+    
+EOF
+{{< /text >}}
+
+Apply the configuration to `cluster2`:
+
+{{< text bash >}}
+$ istioctl install --context="${CTX_CLUSTER2}" -f cluster2.yaml
+{{< /text >}}
+
+{{< /tab >}}
+{{< tab name="Helm" category-value="helm" >}}
+
+Install Istio as primary in `cluster2` using the following Helm commands:
+
+Install the `base` chart in `cluster2`:
+
+{{< text bash >}}
+$ helm install istio-base istio/base -n istio-system --kube-context "${CTX_CLUSTER2}"
+{{< /text >}}
+
+Then, install the `istiod` chart in `cluster2` with the following multi-cluster settings:
+
+{{< text bash >}}
+$ helm install istiod istio/istiod -n istio-system --kube-context "${CTX_CLUSTER2}" --set global.meshID=mesh1 --set global.multiCluster.clusterName=cluster2 --set global.network=network2 --set profile=ambient --set pilot.env.AMBIENT_ENABLE_MULTI_NETWORK="true"
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+## Install the east-west gateway in `cluster2`
```

Comment:
This naming should match the gateway installation in cluster1 https://github.com/istio/istio.io/pull/16709/files#diff-6dd54401d34ad365598b8828ad2d2fb7790338ff205a0b8007704a69e1f647b9R99 and should have the same subcategories